cmd/scepclient/csr.go

```diff
@@ -19,6 +19,7 @@ const (
 
 type csrOptions struct {
 	cn, org, country, ou, locality, province, challenge string
+	cn, org, country, ou, locality, province, dnsnames, challenge string
```

Comment:
```suggestion
	cn, org, country, ou, locality, province, dnsName, challenge string
```

[mixedCaps](https://go.dev/doc/effective_go#mixed-caps) :) also made this singular as its a single string.